tools/codegen/schema/codetemplate/template.go

```diff
@@ -0,0 +1,31 @@
+package codetemplate
+
+import (
+	"bytes"
+	_ "embed"
+	"text/template"
+)
+
+//go:embed schema-file.go.tmpl
+var schemaFileTemplate string
+
+type SchemaFileInputs struct {
+	PackageName      string
+	Imports          []string
+	SchemaAttributes []string
+}
+
+func ApplySchemaFileTemplate(inputs SchemaFileInputs) bytes.Buffer {
+	t, err := template.New("template").Parse(schemaFileTemplate)
```

Comment:
maybe `template` -> `schema_go`

Reply:
Found no value in the name that need to be passed when creating the template, so decide to leave generic. Likely after having other templates this will be extracted to a common function for applying templates.